test/check-lorax

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+from cockpittest import main
+from loraxtest import LoraxTestCase
+
+
+class TestLorax(LoraxTestCase):
+    """
+        This is the "entry-point" to the test suite when
+        executed in Cockpit CI.
+    """
+    def test_lorax_bootiso(self):
+        self.runLoraxTest("/tests/lorax/test_lorax_bootiso.sh")
```

Comment:
This is modelled after the `tests/cli` tests, which is quite the complex architecture, because it mixes two programming languages and test programs that run outside and inside the VM. I did it this way because I wanted to re-use the existing beakerlib scripts.

Beakerlib doesn't lend itself very well to what we're doing (for example, the ugly hack we have to do to make it stop on first failure).

This approach also requires a rebuild of the test vm every time the test script is changed.

I'd prefer having no extra scripts on the VM and doing all testing from the outside. Do you think that's feasible?

Reply:
Maybe? Right now I just want to get it running. We can optimize later. Copying the existing setup seemed like the best way to start.

Reply:
Of course, it was probably easier to get going based on the existing code. It shouldn't be too hard to port over later.